packages/fuels-test-helpers/src/chains.rs

```diff
@@ -1,46 +1,21 @@
 use crate::utils::retry;
 use anyhow::bail;
-use fuel_types::AssetId;
-use fuels_core::constants::{BASE_ASSET_ID, DEFAULT_SPENDABLE_COIN_AMOUNT};
-use fuels_core::tx::{Input, Output, Transaction};
 use fuels_signers::provider::Provider;
-use fuels_signers::{LocalWallet, Signer};
 use fuels_types::errors::Error;
-use rand::Rng;
 use std::time::Duration;
 
-/// Can be used to produce `amount` number of blocks. A block is produced by
-/// submitting a no-op script for execution. Transactions are submitted one
-/// after the other. A new transaction won't be submitted until an increase of
-/// block height has been noticed.
-///
-/// # Arguments
-///
-/// * `wallet`: A `Wallet` that must contain a provider and spendable coins.
-/// * `amount`: By how much to increase the block height.
-///
-/// returns: Result<(), Error>
-pub async fn produce_blocks(wallet: &LocalWallet, amount: usize) -> Result<(), Error> {
-    let provider = wallet.get_provider()?;
-    for _ in 0..amount {
-        let height_before_transaction = provider.latest_block_height().await?;
-
-        let transaction = no_op_signed_transaction(wallet).await?;
-
-        provider.send_transaction(&transaction).await?;
-
-        confirm_block_created(provider, height_before_transaction).await?;
-    }
-    Ok(())
-}
-
-async fn confirm_block_created(provider: &Provider, previous_height: u64) -> Result<(), Error> {
+pub async fn confirm_blocks_created(
```

Comment:
Since we don't use a no op script anymore, this error message is inaccurate:
**"Couldn't confirm a block generation via no-op script -- {}"**